scripts/upgrade/state/state-upgrade-2.ts

```diff
@@ -0,0 +1,45 @@
+import { DeployHelper } from "../../../helpers/DeployHelper";
+import { ethers } from "hardhat";
+
+
+const proxyAdminAddress = "0x4862E19501BdDF875838b7780c2dcEa1713774f8";
+const stateContractAddress = "0xF03116F78Ce803a778979B288A6620e9DC703A17";
+
+const stateLibAddress = "0xF86f066441c43039Ebf73dDB694Da9AB9C5e49F8";
+const smtLibAddress = "0x93772dDfebBF46670577FbeFDe7D80D8C8487796";
+const poseidonUnit1LAddress = "0xA07803BE24640292FA21Ab28B12760B3DA8026f2";
+const verifierAddress = "0xa6B209675817033d98BD564C085566c773f7C7EC";
+
+
+async function main() {
```

Comment:
I think we don't need this file if we upgrade through zeppelin

Reply:
removed